src/modules/ContentList/index.tsx

```diff
@@ -120,11 +125,12 @@ export function ContentListUI(props: ContentListUIProps): ReactElement {
                             .with({ status: 'paused' }, () => wrapMsg('There is no content.'))
                             .with({ status: 'error' }, () => wrapMsg('There was an error loading content.'))
                             .with({ status: 'empty' }, () => wrapMsg('There is no content.'))
-                            .with({ status: 'success' }, ({ data: { content } }) => (
+                            .with({ status: 'success' }, ({ data: { search } }) => (
                                 <ul className="w-full">
-                                    {content.map((item) => {
-                                        const primaryQuote = getPrimaryQuote(item.primaryCompany);
-                                        const date = DateTime.fromISO(item.publishedDate);
+                                    {(search as Search).content.hits.map((hit: ContentSearchResultHit) => {
```

Comment:
Shouldn't need the cast per my comment above, this should already have the right type based on the query.